src/services/email.service.js

```diff
@@ -0,0 +1,69 @@
+require('dotenv/config');
+
+const nodemailer = require('nodemailer');
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASS,
+  },
+});
+
+const send = ({ email, subject, html }) => {
+  return transporter.sendMail({
+    to: email,
+    subject,
+    html,
+  });
```

Comment:
It would be beneficial to add error handling for the send function to catch and log any errors that occur during the email sending process.